packages/api/src/router/test.ts

```diff
@@ -224,6 +224,51 @@ export const testRouter = router({
 
       const userId = ctx.auth.userId;
 
+      const isUserPremium = await ctx.prisma.user
+        .findUnique({
+          where: {
+            userId,
+          },
+          select: {
+            isPremium: true,
+          },
+        })
+        .then((user) => user?.isPremium);
+
+      const userTestCount = await ctx.prisma.test.count({
+        where: {
+          userId,
+        },
+      });
+
+      if (!isUserPremium && userTestCount >= 10) {
+        throw new TRPCError({
+          code: "UNAUTHORIZED",
+          message: "Create a premium account to create more tests",
```

Comment:
Can you change this to:

"Subscribe to our premium to create more tests"

Reply:
got it